tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -70,7 +70,7 @@ def test_incorrect_use_case_id_value(self):
         assert response.status_code == 400
         assert (
             response.json()["detail"]
-            == "Invalid useCase parameter. Please use one of: ['release-health', 'performance']"
+            == f"Invalid useCase parameter. Please use one of: {[uc.value for uc in UseCaseID]}"
```

Comment:
(as commented in the endpoint)

I don't think we should change the inputs of endpoints at this time.
If we decide to do it 
* we should first accept both UseCaseKey and UseCaseID in the endpoints
* change the UI (second PR)
* change the endpoints to accept only UseCaseID ( 3rd PR )